src/webhooks/pubsub/slackScores.test.ts

```diff
@@ -45,67 +67,67 @@ describe('slackScores Tests', function () {
 | null                          |   - (0/0)      |
 | ospo                          |   - (0/0)      |
```

Comment:
Does this mean we always report ospo, even when no metrics are returned? Or is that just how the test is configured?

Reply:
it's just how the test is configured, it uses the teams in `test/product-owners.yml`